app/Exceptions/Handler.php

```diff
@@ -88,7 +88,7 @@ public function render($request, Exception $exception)
             * Usually because user stayed on the same screen too long and their session expired
             */
             if ($exception instanceof UnauthorizedHttpException) {
-                switch (get_class($exception->getPrevious())) {
+                switch (is_object($exception->getPrevious()) && null !== $exception->getPrevious() && get_class($exception->getPrevious())) {
```

Comment:
Is this because get_class in PHP 7.2 does not support 'null' as an argument ? But I don't think we need to check if the object is an object, because in the previous line(90), we are already checking if the $exception is an instance of HttpException, so the getPrevious() will only be either null or an object, we can just check null !== $exception->getPrevious() and remove is_object() check.

```suggestion
                switch ($exception->getPrevious() !== null && get_class($exception->getPrevious())) {
```